README.md

```diff
@@ -17,6 +17,15 @@ Maintainer: Joakim Gross <joakim.gross@pelagicore.com>
 * texlive-latex-extra (when building PDF)
 * texlive-latex-recommended (when building PDF)
 
+###  Install build dependencies on Debian
+
+```
+sudo apt-get install python-pip
+pip install sphinxcontrib-seqdiag sphinxcontrib-blockdiag \
+              sphinxcontrib-actdiag sphinxcontrib-actdiag \
+              sphinxcontrib-manpage
```

Comment:
Yes they are available through apt in debian, at least in debian 9. However, we still have to use pip in other places, for instance vagrant-cookbook, to install packages that are not available through apt so in the interest of consistency we should use pip to install python packages even in this case. Furthermore, pip will provide you with a more up to date version.